tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -269,8 +271,11 @@ def config_mirror(self, peer_cluster, **kw):
         else:
             log.error("Peers were not added")
 
-        self.wait_for_status(poolname=poolname, health_pattern="OK")
-        peer_cluster.wait_for_status(poolname=poolname, health_pattern="OK")
+        # Waiting for OK pool mirror status to be okay based on user input as in image based
+        # mirorring mirroring status wouldn't reach OK without enabling mirroing on individual images
```

Comment:
mirroring word gets duplicated.

Reply:
Updated, please check
Sorry for the silly miss and thanks for pointing out